### PR TITLE
Tania feedback updates

### PR DESCRIPTION
**Pull request by djbrooke**
Some more controversial stuff in here than my usual PRs. I'll be testing some of this locally in parallel.

### CODE REVIEW

File: data/dataverses/manchester/datasets/test-dataset/test-dataset.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
             "typeName": "title",
             "multiple": false,
             "typeClass": "primitive",
-            "value": "Test Dataset"
+            "value": "Test Dataset with Shapefiles, Images, and Tabular Files"
```
Comment (pdurbin) [inline data/dataverses/manchester/datasets/test-dataset/test-dataset.json]: I would consider removing the word "test" entirely.

File: ec2config.yaml [hunk 2]
```diff
@@ -111,6 +109,8 @@ dataverse:
   version: 4.15.1
   wholetale:
     enabled: true
+  dataexplorer:
+    enabled: true
```
Comment (pdurbin) [inline ec2config.yaml]: Interesting! I hope this file is working for you!

Comment (djbrooke) [reply ec2config.yaml]: I updated it on my local copy, destroyed all the data, and now running create_sample_data.py, so more info soon, hopefully.

File: data/dataverses/manchester/datasets/test-dataset/test-dataset.json (hunk 1 above)
Comment (djbrooke) [reply data/dataverses/manchester/datasets/test-dataset/test-dataset.json]: Done

File: ec2config.yaml (hunk 2 above)
Comment (djbrooke) [reply ec2config.yaml]: It worked! I get both options here and they each open in their respective tool.

http://ec2-3-80-234-144.compute-1.amazonaws.com/file.xhtml?fileId=255&version=1.0

Hooray!

**Pull request merged**